### PR TITLE
Include Dan's historical data

### PR DESCRIPTION
**Pull request by dereklieu**
Generate a csv of results based on the data Dan sent with minor modifications to the `ap-to-csv` script.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1190,3 +1189,10 @@ shp/ks/congress-ungrouped.shp: shp/us/congress-ungrouped.shp
 	mkdir -p $(dir $@)
 	ogr2ogr -f 'ESRI Shapefile' -where "STATEFP = '20'" $(dir $@) $<
 
+#
+# Historical JSON
+#
+data/simplified-historical.json: data/historical.json
+	cat data/historical.json \
+		| ./simplify-historical \
+		> $@
```
Comment (dereklieu) [inline Makefile]: Simplifying the candidate structure seems like low hanging fruit here. Gets the before-gzip size to 1.7mb, after gzip to 160k.

Comment (anandthakker) [inline Makefile]: 👍

File: Makefile [hunk 2]
```diff
@@ -990,29 +989,29 @@ geojson/albers/%.geojson: geojson/%.geojson
 geojson/albers/state-bounds.json: geojson/albers/states.geojson
 	cat $^ | ./extract-projected-bounds > $@
 
-election-results/2012.csv:
+election-results/historical.csv:
 	mkdir -p $(dir $@)
-	data/ap-to-csv data/2012-county-results.json data/2012-district-results.json > $@
+	data/ap-to-csv data/historical.json > $@
 
 election-results/2012-state-centroids.geojson: geojson/albers/state-centroids.geojson
```
Comment (anandthakker) [inline Makefile]: We should also change the names of these files to `historical-` instead of `2012-`

File: Makefile [hunk 3]
```diff
@@ -986,29 +987,36 @@ geojson/albers/%.geojson: geojson/%.geojson
 		| ./normalize-properties GEOID:id STATE_FIPS:id FIPS:id  \
 		> $@
 
+geojson/albers/us-10m/counties.geojson: geojson/counties.geojson
+	mkdir -p $(dir $@)
+	cat $^ \
+		| ./reproject-geojson \
+		| ./normalize-properties GEOID:id STATE_FIPS:id FIPS:id  \
+		> $@
+
```
Comment (dereklieu) [inline Makefile:996]: @anandthakker had to make this very specific to overrule an earlier make target that specified `geojson/%/counties.geojson`, which was causing an error.

Comment (anandthakker) [inline Makefile:996]: Huh.  Strange that `states` didn't have this problem, but anyway good catch

**Comment by anandthakker** [comment]
@dereklieu Moved `simplify-historical` -- see https://github.com/developmentseed/wapo-components/pull/1177

**Pull request merged**